signingscript/src/signingscript/sign.py

```diff
@@ -1328,20 +1334,19 @@ async def signer(digest, digest_algo):
         log.info("Not using specified comment to sign %s, not yet implemented for non *.msi files.", orig_path)
         authenticode_comment = None
 
-    if not await winsign.sign.sign_file(
-        infile,
-        outfile,
-        digest_algo,
-        certs,
-        signer,
-        cafile=cafile,
-        timestampfile=timestampfile,
-        url=url,
-        comment=authenticode_comment,
-        crosscert=crosscert,
-        timestamp_style=timestamp_style,
-    ):
-        raise IOError(f"Couldn't sign {orig_path}")
+    # Retry winsign.sign.sign_file, because the timestamp server can hiccup
+    await retry_async(
```

Comment:
You are happy with default args for retry_async attempts, sleeptime, etc? (Just checking - not suggesting a change.)

Reply:
Yeah, I'm not sure which retry args to specify, and I generally default to the default unless I have an opinion.